docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -4,15 +4,15 @@
 
 **Category**: Convolution
 
-**Short description**: Computes the gradients of a Convolution operation with respect to the input. Also known as a Deconvolution or a Transposed Convolution.
+**Short description**: Computes 1D, 2D or 3D *ConvolutionBackpropData* operation with respect to the input and kernel tensors. Also known as a Transposed Convolution.
```

Comment:
Why did you remove mentioning of "Deconvolution"?

Reply:
Deconvolution is a mathematical operation which reverts process of convolution (on the output produces original image). Transposed convolution only reconstructs the spatial dimensions of the image, but output data is not the same as on the input. I removed Deconvolution term as it not exactly fits to this op.

Reply:
@lazarevevgeny, please close this discussion if you are ok with my response.